### ISSUE TITLE
Tflint ignores "--format junit" option

### ISSUE THREAD

**Issue by vorkos** [terraform-linters/tflint#842]
Tflint, built from the last commit https://github.com/terraform-linters/tflint/commit/c72fbe269db59807682e713ba2a9a170c4ace7db ignores "--format junit"  on findings. It works well when there are no findings.

```hcl
module "vpc" {
//...
    enable_dynamodb_endpoint    = true 
    enable_dns_hostnames        = true
    enable_dynamodb_endpoint    = true
//...
}
```
Output - 
" example/main.tf:13,5-29: Attribute redefined; The argument "enable_dynamodb_endpoint" was already set at example/main.tf:11,5-29. Each argument may be set only once." 

but I'm expecting a junit format.

## Version

https://github.com/terraform-linters/tflint/commit/c72fbe269db59807682e713ba2a9a170c4ace7db

**Comment by bendrucker** [comment]
This is not currently supported, https://github.com/terraform-linters/tflint/issues/786 is tracking the request for this

**Comment by bendrucker** [comment]
Can you post the exact command you ran? This should fail:

```
$ tflint --format junit
Failed to parse CLI options. An error occurred:

Error: Invalid value `junit' for option `-f, --format'. Allowed values are: default, json or checkstyle
```

**Comment by vorkos** [comment]
Sorry, my bad - I mixed up tfsec and tflint
